### PR TITLE
Added a prop to support custom options in the dropdown.

### PR DESCRIPTION
**Pull request by shishiranshuman**
I have added an example on the demo page as well. @benkeen Request you to have a look and let me know if any improvements are needed.
Also if you don't feel like adding this feature, please feel free to close the PR.

react_country_region_selector

screen_shot_2018-01-22_at_4 22 41_pm

react_country_region_selector-2

### CODE REVIEW

File: examples/src/examples.js [hunk 1]
```diff
@@ -235,6 +256,8 @@ class ExamplesPage extends React.Component {
   }
 
   selectRegion (exampleIndex, val) {
+      console.log('val', val)
+      console.log('exampleIndex', exampleIndex)
```
Comment (benkeen) [inline examples/src/examples.js]: Could you remove the console.log's?

File: package.json [hunk 2]
```diff
@@ -43,7 +43,8 @@
   },
   "dependencies": {
     "prop-types": "^15.5.8",
-    "reactify": "^1.1.1"
+    "reactify": "^1.1.1",
+    "require-dir": "0.3.2"
```
Comment (benkeen) [inline package.json]: Is this needed? If so, could it be in `devDependencies`?

**Comment by benkeen** [comment]
Nice! Yeah I can imagine this being useful. Just a couple of minor comments listed above.

File: examples/src/examples.js (hunk 1 above)
Comment (shishiranshuman) [reply examples/src/examples.js]: Done

File: package.json (hunk 2 above)
Comment (shishiranshuman) [reply package.json]: It was not needed, I have removed it.
I had forgotten to remove it before creating the PR.
Thanks.

**Comment by shishiranshuman** [comment]
@benkeen Hi, I just wanted to know if anything is blocking or wrong with PR. 🙂

**Comment by shishiranshuman** [comment]
Hi @benkeen, I am closing this PR for now. I'll create a new one with these changes once the v1.4.0 is released. 
I hope you are okay with this.

**Comment by benkeen** [comment]
Ack, sorry I missed this @shishiranshuman! Yes, maybe wait for 1.4 - it's getting pretty close, just a few more days.

**Pull request closed without merging**